unit/src/MooseServerTest.C

```diff
@@ -1571,51 +1573,72 @@ TEST_F(MooseServerTest, DocumentReferencesRequest)
 
 TEST_F(MooseServerTest, DocumentFormattingRequest)
 {
+  // put variables file on disk which will be included from base input file
+
+  std::ofstream include_variables("include_variables.i");
```

Comment:
why not commit that as a file?

Reply:
The base input is not a file on disk and just gets passed as a `std::string` to the server.

And that base input simply specifies `!include include_variables.i` to include this file.

So it think it is straightforward to put the file in the test working directory then remove it.

This way it just always works as that file will be in whichever directory the test is running.

Otherwise I'd need to think about paths to its location from the testing working directory.

It also may muddle the new feature that finds and removes input files that are not tested.

This is the same way we are doing the actual `!include` unit testing in [`HitTests.C`](https://github.com/idaholab/moose/blob/e04b928f769ca6cabbdbe635fb823e379a2bc8ad/unit/src/HitTests.C#L806) also.